Src/java/qdm/src/main/resources/gov/healthit/qdm/qdm-modelinfo-5.3.xml

```diff
@@ -85,8 +85,8 @@
         <ns4:element name="admissionSource" type="System.Code"/>
         <ns4:element name="relevantPeriod" type="interval&lt;System.DateTime&gt;"/>
         <ns4:element name="dischargeDisposition" type="System.Code"/>
-        <ns4:element name="facilityLocation" type="QDM.FacilityLocation"/>
-        <ns4:element name="diagnosis" type="list&lt;System.Code&gt;"/>
+        <ns4:element name="facilityLocations" type="QDM.FacilityLocation"/>
+        <ns4:element name="diagnoses" type="list&lt;System.Code&gt;"/>
```

Comment:
I intentionally left that change out of the schema because from an XML perspective, it's still an "element", it's just that the element may appear more than once, so that's more consistent with conventions in an XSD file.